src/session-pool.ts

```diff
@@ -157,9 +158,8 @@ const DEFAULTS: SessionPoolOptions = {
  * Error to be thrown when attempting to release unknown resources.
  */
 export class ReleaseError extends Error {
-  // tslint:disable-next-line no-any
-  resource: any;
-  constructor(resource) {
+  resource: Any;
```

Comment:
We could probably just use `object` in place of `any`.

Reply:
Actually, can we use `unknown` here? Seems fitting, haha.

Reply:
Respect - fixed!